ci/scripts/run_pytest.sh

```diff
@@ -23,17 +23,33 @@ export BIONEMO_DATA_SOURCE="${BIONEMO_DATA_SOURCE:-pbss}"
 export PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True
 source "$(dirname "$0")/utils.sh"
 
+# Parse options
+USE_NBVAL=true
+for arg in "$@"; do
+    case $arg in
+        --no-nbval)
+            USE_NBVAL=false
+            shift
+            ;;
+    esac
+done
+
 if ! set_bionemo_home; then
     exit 1
 fi
 
 python -m coverage erase
 
 error=false
+# Build pytest options dynamically
+pytest_options="-v --durations=0 --durations-min=60.0"
+if [ "$USE_NBVAL" = true ]; then
+    pytest_options="$pytest_options --nbval-lax"
+fi
 for dir in docs/ ./sub-packages/bionemo-*/; do
```

Comment:
I think the better approach might be to just not run pytest in the `docs/` directory?
That might give us a better separation of "example notebooks" that we don't want to run in pre-merge testing, vs. testing functionality that we expect to work inside a jupyter notebook.

Reply:
good point, replaced --no-nbval flag with --skip-docs